### DISCUSSION TITLE
The issue when try to start a tpu-vm

### DISCUSSION THREAD

**Discussion by sglucas** [jax-ml/jax#16485] [General]
Hi @skye 

When I try to start a tpu-vm v3-32 with this command:

`gcloud alpha compute tpus tpu-vm create v3-32    --zone europe-west4-a --accelerator-type v3-32  --version v2-alpha`

it occurs this error:

`ERROR: (gcloud.alpha.compute.tpus.tpu-vm.create) {
  "code": 9,
  "message": "Insufficient reserved capacity. Contact customer support to increase your reservation. [EID: 0x72a13ef4f9caeaa2]"
}`

However I can successfully start a tpu-vm v3-128.

It could be better if you can give me some advice

Best 
Lucas

**Comment by hawkinsp** [comment]
I note this issue isn't related to JAX specifically and would probably be best addressed through Cloud TPU support channels.

**Reply by sglucas** [reply]
Thanks for your reply.

May I ask how to contact cloud TPU support?

Best,
Lucas

**Comment by ssusie** [comment]
Hi @sglucas, do you have a reserved capacity or you are using on-demand quota? were you able to create v3-128 right after you failed to create v3-32? This type of error usually occurs when there is no capacity available.

**Reply by sglucas** [reply]
Yes, I can create v3-128 now but v3-32 is still not available. So I think I still have quota to start v3-32.

Best,
Lucas

**Reply by ssusie** [reply]
Did you delete v3-128 before retrying v3-32?  You can reach out to cloud-tpu-support@google.com. In the email please include your project ID, the full error message you get when trying to create the tpu. Hope this helps.

**Comment by ssusie** [comment]
Also please refer to this guide for choosing a version for accelerator type https://cloud.google.com/tpu/docs/supported-tpu-configurations#jax 
` --version v2-alpha` is very old and deprecated.

**Reply by sglucas** [reply]
Thanks for your answer. I try to use other version and I still can not start v3-32.

Best,
Lucas
